CreateProfile/__tests__/handler.test.ts

```diff
@@ -1,5 +1,3 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I removed the linter ignoring comments because the tests file are ignored by eslint.